### PR TITLE
Fix: Documents are not copied completely

### PR DESCRIPTION
**Pull request by blankse**
## Changes in this pull request  
Resolves https://github.com/pimcore/pimcore/issues/17455
Regression from https://github.com/pimcore/pimcore/pull/16010

### CODE REVIEW

File: bundles/InstallBundle/src/Installer.php [hunk 1]
```diff
@@ -798,7 +798,7 @@ protected function insertDatabaseContents(Connection $db): void
     {
         $db->insert('assets', Helper::quoteDataIdentifiers($db, [
             'id' => 1,
-            'parentId' => 0,
+            'parentId' => null,
```
Comment (martineiber) [inline bundles/InstallBundle/src/Installer.php]: @blankse What was the idea behind this change? Why is this necessary?

I would see that as a BC. If someone has a condition on an Listing to get the `parentId = 0` this would not work anymore.

Comment (blankse) [reply bundles/InstallBundle/src/Installer.php]: @martineiber When writing the test, I noticed that the ParentId is 0 after copying. I think it should be consistent with the other properties NULL. But yes changing the ParentIds on the root elements could be a BC break. I'll change that back.

Comment (blankse) [reply bundles/InstallBundle/src/Installer.php]: @martineiber Done :)

File: tests/Model/Asset/AssetTest.php [hunk 2]
```diff
@@ -98,9 +98,9 @@ public function testParentIs0(): void
         $this->expectException(Exception::class);
         $this->expectExceptionMessage('ParentID is mandatory and can´t be null. If you want to add the element as a child to the tree´s root node, consider setting ParentID to 1.');
         $savedObject = TestHelper::createImageAsset('', null, false);
-        $this->assertTrue($savedObject->getId() == 0);
+        $this->assertNull($savedObject->getId());
 
-        $savedObject->setParentId(0);
+        $savedObject->setParentId(null);
```
Comment (martineiber) [inline tests/Model/Asset/AssetTest.php:117]: This should still throw an exception if `$savedObject->setParentId(0);` is called! 

Could you please add this Test case as well?

**Review by martineiber — CHANGES_REQUESTED** [review]

File: tests/Model/Asset/AssetTest.php (hunk 2 above)
Comment (blankse) [reply tests/Model/Asset/AssetTest.php:117]: @martineiber Done https://github.com/pimcore/pimcore/pull/17805/commits/4fef2c073659a11d6007d903afcd5089eb8d6e59

**Review by martineiber — APPROVED** [review]
LGTM :+1: 

Thank you for your help.
